### PR TITLE
Covenant Tweaks

### PR DESCRIPTION
**Pull request by Stingray540**
:cl: Stingray
rscadd: Adds Covenant backpacks, medkits (mapped), and a limited amount of ranger suits
tweak: Modified values of NPC health and ship overmap damage (does not affect pvp) to make them feel more balanced
/:cl:

### CODE REVIEW

File: code/modules/halo/species_items/sangheili.dm [hunk 1]
```diff
@@ -60,6 +72,7 @@ GLOBAL_LIST_INIT(last_names_sangheili, world.file2list('code/modules/halo/specie
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
 	specials = list(/datum/armourspecials/shields,/datum/armourspecials/shieldmonitor/sangheili)
 	armor = list(melee = 85, bullet = 65, laser = 60, energy = 60, bomb = 55, bio = 25, rad = 25) //Close to spartan armour. Lower bullet,higher melee. Lower energy.
+	armor_thickness = 20
```
Comment (X0-11) [inline code/modules/halo/species_items/sangheili.dm]: This is uneeded, all /armor subtypes inherit an armor thickness of 20 from /armor

File: code/modules/halo/overmap/combat_npc_ship.dm [hunk 2]
```diff
@@ -14,7 +14,7 @@
 	name = "Combat Ship"
 	desc=  "A ship specialised for combat."
 
-	hull = 3000 //Hardier than a civvie ship.
+	hull = 500 //Hardier than a civvie ship.
```
Comment (X0-11) [inline code/modules/halo/overmap/combat_npc_ship.dm]: 500 is way too low. The weaker dual MAC setup does 500 per MAC hit and the energy projector does around 1000.
3000 health was chosen to allow combat ships to actually participate.
Combat ships do however not get extra damage from capacitors and as such their main-ship-weapon projectiles do much less damage than player fired ones do.

**Review by X0-11 — DISMISSED** [review]

File: code/modules/halo/overmap/combat_npc_ship.dm (hunk 2 above)
Comment (Stingray540) [reply code/modules/halo/overmap/combat_npc_ship.dm]: Discussed on discord and changed with my recent commit.

File: code/modules/halo/species_items/sangheili.dm (hunk 1 above)
Comment (Stingray540) [reply code/modules/halo/species_items/sangheili.dm]: Removed.

**Review by BDpuffy420 — APPROVED** [review]

**Pull request merged**